### PR TITLE
Added missing endpoint getDatapointsByName

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by valgussev — DISMISSED** [review]
lgtm!

File: src/__tests__/Datapoints.test.ts [hunk 1]
```diff
@@ -105,6 +105,38 @@ describe('Datapoints', () => {
     });
   });
 
+  test('retrive datapoints by name', async () => {
```
Comment (martincognite) [inline src/__tests__/Datapoints.test.ts]: nit; typo

**Review by martincognite — DISMISSED** [review]

File: src/__tests__/Datapoints.test.ts (hunk 1 above)
Comment (martincognite) [inline src/__tests__/Datapoints.test.ts]:
```suggestion
  test('retrieve datapoints by name', async () => {
```

**Review by valgussev — APPROVED** [review]

**Review by martincognite — APPROVED** [review]
LGTM

**Pull request merged**